action/src/stoatApiHelpers.ts

```diff
@@ -12,7 +12,7 @@ export type DevServerCheck = false | string;
 export const STOAT_ORG = 'stoat-dev';
 export const STOAT_REPO = 'stoat';
 export const STOAT_ACTION_REPO = 'stoat-action';
-export const INTERNAL_REPOS = [STOAT_REPO, STOAT_ACTION_REPO];
+export const INTERNAL_REPOS = [STOAT_REPO, STOAT_ACTION_REPO, 'd2'];
```

Comment:
nit: we should probably allow this override in the repo we're using, not in a checked-into-the-action location.